### DISCUSSION TITLE
[6주차 알고리즘 스터디] 이진 탐색

### DISCUSSION THREAD

**Discussion by bigsaigon333** [fe-backpackers/js-algorithm-basecamp#64] [General]
# [6주차 알고리즘 스터디] 이진 탐색

## 이번주 문제
- Q28. 고정점 찾기
- [Q29. 공유기 설치](https://www.acmicpc.net/problem/2110)

## 회고 순서
체크인 : ~22:05
다른 사람 코드 참고하여 나의 코드 개선: ~22:20
자유로운 의견교환: ~22:40
개인 회고: ~22:50
팀별 공유: ~23:00

**Comment by bigsaigon333** [comment]
## 체크인

오늘의 기분을 10점 만점으로 이유와 함께 적어주세요~

**Reply by ganeodolu** [reply]
9점 : 간만에 바깥바람을 쐬어서 좋네요ㅎㅎㅎ

**Reply by bigsaigon333** [reply]
6점: 면접 준비, 자소서 작성 으로 바쁘고 괴로운 나날을 보내고 있습니다 :sob:

**Comment by bigsaigon333** [comment]
## 개인 회고

**Reply by bigsaigon333** [reply]
면접준비 등으로 시간이 부족하다는 핑계로 급하게 문제를 풀었네요. 이론은 출퇴근 지하철에서 틈틈히 봤는데 정리를 하는 것도 오래걸렸습니다..
이번주 DP는 꽤나 어려울 것으로 예상되는데, 틈틈히 이론 공부하고 정리도 미리미리 해야겠어요. 

### 이번주 Action Plan
- 화요일: 이론 학습
- 수요일: 이론 정리
- 목요일:  8장 예제 1
- 금요일:  8장 예제 2
- 토요일: 16장 문제풀이 1
- 일요일: 16장 문제풀이 2

**Reply by Sunmon** [reply]
이번주는 개인프로젝트 + 회사일로 바빴고... 11월 20일까지는 그럴 것 같습니다. ㅠㅠ

**Reply by ganeodolu** [reply]
DP가 난도가 있는 것으로 잘 알려져 있어서 시간을 더 투자해야겠다는 생각이 들었습니다.
다음주도 화이팅하겠습니다 !!!

**Comment by bigsaigon333** [comment]
## 체크아웃

**Reply by ganeodolu** [reply]
9점 : 소규모로 수다도 떨면서 이해안되던 29번 문제가 좀 이해되는 것 같아서 끝나고 다시 살펴보겠습니다. 👍🏼

**Reply by Sunmon** [reply]
8점: 사이드 프로젝트에 참여해서 정신이 없네요 @,.@ 사이드플젝 정리되는대로 다시 알고리즘 열심히 참여하겠습니다

**Reply by bigsaigon333** [reply]
8점: 제가 주최한 스터디라서 빠질수가 없어 꾸역꾸역 풀었는데, 이렇게라도 알고리즘 공부를 하게 되어서 정말 다행입니다 ㅎㅎ..
